runtime/runtime-core/common/src/aws/smithy/kotlin/runtime/collections/ValuesMap.kt

```diff
@@ -2,9 +2,10 @@
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  * SPDX-License-Identifier: Apache-2.0
  */
-package aws.smithy.kotlin.runtime.util
+package aws.smithy.kotlin.runtime.collections
 
 import aws.smithy.kotlin.runtime.InternalApi
+import aws.smithy.kotlin.runtime.util.CaseInsensitiveMap
```

Comment:
**fix**: Should this not be in collections?

Also (and this can be a follow on PR) we should look at what's left in util and see if it still belongs there. On surface `Stack` and `ReadThroughCache` utils probably belongs in collections, possibly `Attributes`?. Also some of the other APIs probably belong in `io` or elsewhere.

Reply:
Yes, it should. I can move it and the others you mentioned to **collections** as well.